docs/sources/flow/setup/start-agent.md

```diff
@@ -146,26 +145,26 @@ If you downloaded the standalone binary, you must run the agent from a terminal
 To start Grafana Agent on Linux, macOS, or FreeBSD, run the following command in a terminal window:
 
 ```shell
-AGENT_MODE=flow BINARY_PATH run CONFIG_FILE
+AGENT_MODE=flow BINARY run CONFIG_FILE
 ```
 
 Replace the following:
 
-* `BINARY_PATH`: The path to the extracted binary.
-* `CONFIG_FILE`: The path to a Grafana Agent configuration file.
+* `BINARY`: The Grafana Agent binary file. Include the path to the binary file.
```

Comment:
$0.02: I feel like simplifying BINARY_PATH to BINARY might be more ambiguous, especially since it's expected to be the path to the binary.

I'll leave it up to you to decide what to put here though, just wanted to raise my concern.

Reply:
Is it only the path? or the path plus binary?

Reply:
Well, it's the path based on the working directory, so it could be a full path or a relative path.

Reply:
It's best practice for it to be an absolute path to where the binary is, I guess. 🤷

Reply:
I've reverted this change in the next commit.